src/hooks/useAuthorityLinking/useAuthorityLinking.js

```diff
@@ -34,6 +34,20 @@ const useAuthorityLinking = () => {
     return linkingRules.find(rule => rule.bibField === bibTag && rule.authorityField === authorityTag);
   }, [linkingRules]);
 
+  const removeSubfieldsIfThereAreNotInAuthority = useCallback((bibSubfields, authField, bibTag) => {

```

Comment:
We already have a function that iterates over authority subfields, I think it'll be shorter to add an `else` statement inside `copySubfieldsFromAuthority` here - https://github.com/folio-org/ui-quick-marc/blob/master/src/hooks/useAuthorityLinking/useAuthorityLinking.js#L48
```
else {
  delete bibSubfields[formatSubfieldCode(subfieldCode)];
}
```